client/state/comments/actions.js

```diff
@@ -160,6 +161,11 @@ export function requestPostComments( siteId, postId, status = 'approved' ) {
 	};
 }
 
+export const requestCommentsList = query => ( {
+	type: COMMENTS_LIST_REQUEST,
```

Comment:
I know this is firing but I'm not seeing it in redux dev tools. Do we need to reorder middleware again?